src/gen/lib/Alias.mli

```diff
@@ -1,225 +0,0 @@
-(* Extract ALIAS nodes and assign them globally-unique names.
-
-   What are aliases in a tree-sitter grammar?
```

Comment:
maybe we could save this comment somewhere, in some doc, explaining what they are, why they cause pain for us,
and how we remove them. Same for the _.

Reply:
Yes. I was waiting a bit for things to settle before writing a general readme for the project. I'll fish these comments out of the git history.